addons/crm/views/crm_lead_views.xml

```diff
@@ -1106,8 +1106,7 @@
             <field name="res_model">crm.lead</field>
             <field name="view_mode">list,kanban,graph,pivot,calendar,form,activity</field>
             <field name="view_id" ref="crm_lead_view_list_activities"/>
-            <!-- Ensure that only records with at least one activity, "done" (archived) or not, are fetched. -->
-            <field name="domain">[("activity_ids.active", "in", [True, False])]</field>
+            <field name="domain">[("activity_ids.active", "=", True)]</field>
```

Comment:
Can't we leave the domain empty and let the user pick? (will default to this thanks to active_test anyway)